rflx/model/type_.py

```diff
@@ -25,6 +36,19 @@ def dependencies(self) -> ty.List["Type"]:
         """
         return [self]
 
+    @property
+    def qualified_identifier(self) -> ID:
+        """
+        Return the
+        """
+        # TODO: Handle name collision with `type_.qualified_type_identifier`.
```

Comment:
@treiher How do you think? I found the term `qualify` in the function name `type_.qualified_type_identifier` is slightly different from my usage here...

The former actually means "full identifier path, including `__INTERNAL__::` and `__BUILTIN__::`, which are in fact not accepted by the RecordFlux parser.